### PR TITLE
Replaces parent wand with wand of nothing on Donut

### LINKED ISSUE #49883: Pacifists still can't use Wand of Nothing
## Reproduction:
#49240 should have fixed it yet when I'm on Donut and I try and use the wand of nothing as a pacifist I can't.

### PR DESCRIPTION
**Pull request by cacogen**
## About The Pull Request
Fixes #49883

- Renames prototype wand 'wand' to distinguish it from wand of nothing
- Simplifies prototype wand's description to "You shouldn't have this."
- Renames prototype magic projectile to "bolt" to differentiate it from bolt of nothing

## Why It's Good For The Game

The lizard mime can use the thing

## Changelog
:cl: cacogen
fix: Wand of nothing on the theatre stage on Donut can now be used by pacifists
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/magic/wand.dm [hunk 1]
```diff
@@ -30,6 +30,10 @@
 	..()
 
 /obj/item/gun/magic/wand/afterattack(atom/target, mob/living/user)
+	if(type == /obj/item/gun/magic/wand)
```
Comment (LudwigVonChesterfield) [inline code/modules/projectiles/guns/magic/wand.dm]: I don’t think this should even exist... not in this form, surely.

File: code/modules/projectiles/guns/magic/wand.dm [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /obj/item/gun/magic/wand
-	name = "wand of nothing"
-	desc = "It's not just a stick, it's a MAGIC stick! You shouldn't have this."
+	name = "wand"
+	desc = "You shouldn't have this."
```
Comment (Dennok) [inline code/modules/projectiles/guns/magic/wand.dm]: Maybe name it "primordial wand" or prime
"Father of all wands."

File: code/modules/projectiles/guns/magic/wand.dm [hunk 3]
```diff
@@ -10,6 +10,10 @@
 	var/variable_charges = TRUE
 
 /obj/item/gun/magic/wand/Initialize()
+	if(type == /obj/item/gun/magic/wand)
+		visible_message("<span class='warning'>[src] disappears!</span>")
+		qdel(src)
+		return
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/projectiles/guns/magic/wand.dm]: admins should still be able to spawn the base wand if they want to edit it.

should just remove it, you've already made it obvious from the wand of nothing no need for quirky checks.

Comment (cacogen) [reply code/modules/projectiles/guns/magic/wand.dm]: okay

**Pull request merged**